Automattic/wp-calypso
# Show a message about missing stats

Related to `Tumblr/a8c-dsp/issues/2895`

## Proposed Changes
Show a message about missed stats in Advertising Campaign's Details

## Why are these changes being made?
In the case of missing stats inform a customer about the necessity to wait a bit for a couple of hours or something.

## Testing Instructions
- You need full-fledged DSP working locally (DSP API, WPCOM is sandboxed, etc.),
- Provide your local DSP DB with stats, in order to achieve it, please follow instructions from `Tumblr/a8c-dsp/pull/2793`:
  - Create a campaign and change it's ID to become `110111`,
  - Import data from the mentioned PR,
  - Make sure that calling `http://localhost:3003/api/v1.1/stats/110111?start_date=2024-08-23&end_date=2024-08-27&tz=Europe/London` returns data for you,
  - Adjust start_date and end_date in DSP DB for your campaign (end_date in the future, status is `active`),
- Open campaign details for a campaign with stats (`110111`), it should work as usual
- (Actual testing begins)
- Now create a new campaign from Calypso,
- Change the number of impressions for this campaign from `0` a arbitrary positive number,
- Make sure that campaign details shows according sign twice (instead of both graphs).

- For the campaign with stats:
<img width="673" alt="image" src="https://github.com/user-attachments/assets/0ff5cd7c-e057-4ab1-8b94-e81c3351a70f" />

- and for weekly spending:
<img width="674" alt="image" src="https://github.com/user-attachments/assets/45e3883f-fb9b-4043-8207-c33cb3a81e0f" />

- By contrast, the campaign without stats should look like the following:
<img width="677" alt="image" src="https://github.com/user-attachments/assets/33b7524d-edbb-43ed-9d2a-961e2d073d56" />


## Pre-merge Checklist
